chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -92,6 +96,24 @@ public UserManagementController(UserManager userManager, InvitationManager invit
         this.authorizationLengthInDays = authorizationLengthInDays;
     }
 
+    @Operation(summary = "Update the currently logged in user with an additional organization.",
+            description = "Gives the user permission on the object in the invitation (usually an additional ACB or Developer)."
```

Comment:
I think this documentation is copied from the CHPL-login workflow methods. The line about which endpoints to call in which order is not right for the Cognito methods. Can you check the other Cognito auth methods as well? They are not currently in the diff but it looks like they might have the same documentation.